packages/manager/modules/manager-pci-common/src/components/deployment-mode-card/style.scss

```diff
@@ -0,0 +1,34 @@
+.deployment-mode-card {
+  @apply border-[--ods-color-blue-100];
+  @apply w-full px-[24px] py-[16px];
+  min-height: 200px;
+
+  .deployment-mode-card__content {
+    @apply flex flex-col w-full items-center;
+  }
+
+  .deployment-mode-card__header {
+    @apply mb-4;
```

Comment:
shouldn't you use tailwind classes directly, if not you are kind of losing the advantage of using tailwind

Reply:
Tailwind can't handle css selectors so we need to have css, so either I duplicate everything that tailwind is doing in CSS, or I have a mix of tailwind inside react components and other styles inside css